docs/reference/docs/reindex.asciidoc

```diff
@@ -92,6 +92,9 @@ privilege for the source data stream, index, or alias.
 in the `reindex.remote.whitelist` setting of `elasticsearch.yml`. See
 <<reindex-from-remote>>.
 
+* If ccs-based reindexing from a remote cluster, you must add the remote cluster setting. See
```

Comment:
Since it looks like a condition for performing an [existing reindex](https://github.com/elastic/elasticsearch/blob/2895f1e900b2f41704fd507845102a281cff437e/docs/reference/docs/reindex.asciidoc?plain=1#L91), I think it would be better to give it a different name to avoid confusing users.

Reply:
Yes I agree that we should keep the terminology to "ccs-based reindexing" to distinguish this from the pre-existing functionality of reindexing from a remote cluster.